public/app/dsp-bootstrap.js

```
@@ -189,59 +190,12 @@
-          ringCapacity: 16,
-          quantumSize:  128,
-        });
-        window._vipMlWorker = mlWorker;
-      } catch (e) {
-        console.warn('[DSP-Bootstrap] ML worker failed to start:', e);
-      }
+      /* ── AudioWorklet + ML Worker ── */
+      // Worklet registration and worker spawning belong exclusively to
+      // pipeline-orchestrator.js (CLAUDE.md §2 + §3).
+      // dsp-bootstrap wires the audio graph but defers lifecycle to the orchestrator.
+      let workletNode = window._vipWorkletNode || null;
```

ESLint

[error] 'window' is not defined.

(no-undef)